opentok/opentok-meet
# OPENTOK-47710: disable transcription when not a moderator

#### What is this PR doing?
Ensure that only moderators can make a session level transcription start call

#### How should this be manually tested?
Test on moderator and make sure captions still work. Try with publisher and ensure that the button is grey and causes a popup when pressed

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-47710
